### PR TITLE
[WebProfilerBundle] Fix cursor on link that has no href

### PR DESCRIPTION
**Pull request by PhilETaylor**
[WebProfilerBundle] Fix cursor on link that has no href

link to button

| Q             | A
| ------------- | ---
| Branch?       | 6.3 
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 

Alternative approach for https://github.com/symfony/symfony/pull/50376 

@stof @nicolas-grekas

### CODE REVIEW

**Review by nicolas-grekas — COMMENTED** [review]
thanks. can you please rebase to get ird of the merge commit?

**Comment by PhilETaylor** [comment]
I screwed up with rebase so I redid and force pushed, is that ok?

Note the change at the end of profiler.css.twig is correct, as it cleans up one of the 2 blank lines at the end of the file

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig [hunk 1]
```diff
@@ -1541,6 +1541,13 @@ tr.status-warning td {
 
 {# Toggles
    ========================================================================= #}
+.sf-toggle-button {
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig]: We already have `btn-link` implementing that behavior of buttons looking like a link (and with the proper hover state)

**Review by stof — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig (hunk 1 above)
Comment (PhilETaylor) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig]: Ah ok, third time lucky then, will redo when back at my desk. Sorry for the noise. Sometimes the simple changes take the longest to get right - for me.

Comment (PhilETaylor) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig]: I have changed and force pushed again - thanks.

**Comment by PhilETaylor** [comment]
using `.btn-link` now: 


https://github.com/symfony/symfony/assets/400092/9a920f92-a366-4f7f-b0c7-386467662778

**Comment by nicolas-grekas** [comment]
Thank you @PhilETaylor.

**Pull request merged**